readthedocs/templates/projects/project_version_list.html

```diff
@@ -134,8 +135,19 @@ <h1>{% trans "Activate a version" %}</h1>
           </div>
         {% endif %}
 
+
+
       </div>
     </div>
+
+    {% if inactive_versions|length >= max_inactive_versions %}
```

Comment:
> Technically, if the project has exactly 100 versions the message is shown even if nothing was truncated but I think that is acceptable in this instance.

If you use `>` instead of `>=` don't you get the behavior you want?

Reply:
Won't this `inactive_versions|length` always return 100 if there is more than 100 versions?
as we pass this to the context `inactive_versions[:max_inactive_versions]`